src/components/organisms/ItemForm/ItemFormContainer.jsx

```diff
@@ -23,12 +23,14 @@ export default compose(
       const regex = /[|&;$%@"<>()+#,' ']/
       if (regex.test(value)) {
         setFileNameError('* Please avoid using special characters or spaces.')
+      } else if (!value) {
+        setFileNameError('* A file name is required.')
       }
       return sanitizeFileName(value)
     },
     cancelRename: ({ toggleNameModal, setRenameFile, match, editFileName }) => () => {
       editFileName({ target: { value: match['params']['item'] } })
-      toggleNameModal()
+      toggleNameModal(false)
```

Comment:
@tbrandle this won't work for hardcode set to false, since it's checking `payload || !isNameModalOpen` ... we should either update that state handler, or create two (one for "set modal" with either true/false, and the other to toggle)